client/landing/jetpack-cloud/components/sidebar/index.jsx

```diff
@@ -210,11 +217,22 @@ export default connect(
 		const isBackupSectionOpen = isSidebarSectionOpen( state, SIDEBAR_SECTION_BACKUP );
 		const isScanSectionOpen = isSidebarSectionOpen( state, SIDEBAR_SECTION_SCAN );
 		const threats = getSiteThreats( state );
+		const siteId = getSelectedSiteId( state );
+		const rewindState = getRewindState( state, siteId );
+		const rewindStateRequestStatus = getRewindStateRequestStatus( state, siteId );
+
+		const settingsIssues = [];
+
+		if ( rewindStateRequestStatus === 'success' && rewindState?.state !== 'active' ) {
```

Comment:
This logic seems to be used in a few different places. Can we make a helper for it?